modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/BootstrapService.scala

```diff
@@ -188,6 +189,8 @@ object BootstrapService {
         "published",
         "deprecated",
         "links",
+        "rel",
```

Comment:
`rel` and `href` are not used anymore when changing to an object based `links`

Reply:
we should add `nxv:rev`, `nxv:deprecated` and `nxv:published` and the ones related to search

Reply:
```
 List(
        "@context",
        "@id",
        "@type",
        "total",
        "",
        "nxv:rev",
        "nxv:published",
        "nxv:deprecated",
        "links",
      )
```